modules/utxo-lib/src/bitgo/wallet/Psbt.ts

```diff
@@ -7,31 +8,58 @@ import { createPsbtFromTransaction } from '../transaction';
 import { BIP32Interface } from 'bip32';
 import { isWalletUnspent, WalletUnspent } from './Unspent';
 import { checkForInput } from 'bip174/src/lib/utils';
-import { PsbtInput } from 'bip174/src/lib/interfaces';
+import { PartialSig, PsbtInput, TapScriptSig } from 'bip174/src/lib/interfaces';
 import {
   getLeafVersion,
   calculateScriptPathLevel,
   isValidControlBock,
   ParsedPubScriptP2ms,
   ParsedPubScriptTaprootScriptPath,
   parsePubScript,
+  ParsedPubScriptTaproot,
+  ParsedPubScriptTaprootKeyPath,
   ParsedScriptType2Of3,
 } from '../parseInput';
+import { parsePsbtMusig2PartialSigs } from '../Musig2';
+import { isTuple } from '../types';
+import { createTaprootOutputScript } from '../../taproot';
 
-type Signatures = [Buffer] | [Buffer, Buffer] | undefined;
+export interface Signatures {
+  signatures: [Buffer] | [Buffer, Buffer] | undefined;
+}
 
-export interface ParsedPsbt2Of3 extends ParsedPubScriptP2ms {
-  signatures: Signatures;
+/**
+ * To hold parsed signature data for TaprootKeyPathSpend script type.
+ * signatures and participantPublicKeys (signer pub keys)
+ */
+export interface TaprootKeyPathSignatures {
+  signatures: [Buffer] | [Buffer, Buffer] | undefined;
```

Comment:
If I understand this correctly
```suggestion
  /** Two signatures for the pre-aggregated state */ 
  signatures: [Buffer] | [Buffer, Buffer] | undefined;
```

Reply:
Done